src/main/java/gov/nasa/jpl/mbee/mdk/mms/validation/BranchValidator.java

```diff
@@ -104,7 +104,7 @@ public void validate(ProgressStatus progressStatus, boolean allBranches) {
             branchJson.remove(MDKConstants.PARENT_REF_ID_KEY);
             JsonNode value;
             String entryKey;
-            if ((value = branchJson.get(MDKConstants.ID_KEY)) != null && value.isTextual()) {
+            if ((value = branchJson.get(MDKConstants.NAME_KEY)) != null && value.isTextual()) {
```

Comment:
Instead of changing the ID mapping to a NAME mapping, change the other NAME mapping to an ID mapping. Result should be an ID to ID comparison.